lib/_http_server.js

```diff
@@ -55,8 +61,11 @@ const {
   DTRACE_HTTP_SERVER_REQUEST,
   DTRACE_HTTP_SERVER_RESPONSE
 } = require('internal/dtrace');
+const { observerCounts } = internalBinding('performance');
 
 const kServerResponse = Symbol('ServerResponse');
+const kServerResponseStatistics = Symbol('ServerResponseStatistics');
+const IDX_PERFORMANCE_ENTRYTYPE_HTTP = 6;
```

Comment:
This seems somewhat brittle. Is there maybe an alternative that does not require this magic number and that would be able to handle changed entry type positions?

Reply:
I through it's index wasn't exposed through the binding at first (because it's an hidden constant), but actually is it. I've updated that too